### ISSUE TITLE
installation instructions

### ISSUE THREAD

**Issue by justushache** [TheAomx/HP-625-Linux-Fancontrol#1]
I am very thankful that there is someone doing the things hp should do
however it would be nice to have some installation instructions or some other Wiki entry's 
i am hoping that im getting this installed because the fans are really annoying

**Comment by TheAomx** [comment]
Hello,
if you want to use my programm then you first have to compile it using cmake and g++. How you can install these programs depends on the linux distrubution you are using. Assuming that you are using debian or ubuntu you can issue the command 

> sudo apt install build-essential cmake

 to install both packages on your system.

Then clone the repository with git. Open a terminal window and then insert there 

> git clone https://github.com/TheAomx/HP-625-Linux-Fancontrol.git

Get into the cloned directory.

> cd HP-625-Linux-Fancontrol

and then use mkdir to create a build directory.

> mkdir build

 Then use cmake to generate the Makefile.

> cmake ..

After that use make to compile the program.

> make

You can use make install to copy the programm to the folder /opt if you want.

> sudo make install

Then you can run the program from the terminal now.

> sudo /opt/FanControl

Now your fans of your HP 625 should be controlled by the program. If you dont want to always manually start the program, you can use the included systemd service fancontrol.service to start the programm at the system startup. If you want to do that just issue the following commands in a terminal.

> cp ./fancontrol.service /etc/systemd/system/
> sudo systemctl enable fancontrol.service
> sudo systemctl start fancontrol.service

I hope this helps you with your issues installing my fan control program for the hp 625.

**Comment by justushache** [comment]
thank you very much i have searched houres to find an solution to this stupid fan problem. The fans were always shut off an only once the cpu hit 85°C the fans turned on but then to 100% and only shut angain off once the cpu was under 75°C
